src/renderer/components/ft-input-tags/ft-input-tags.vue

```diff
@@ -21,7 +21,24 @@
       :force-action-button-icon-name="['fas', 'arrow-right']"
       @click="updateTags"
     />
-    <div class="ft-tag-box">
+    <div
+      v-if="tagList.length >= 1"
+      class="checkbox-container"
+    >
+      <input
+        :id="label"
+        type="checkbox"
+        :checked="showTags"
+        @change="toggleShowTags"
+      >
+      <label :for="label">
```

Comment:
Please find a way to make this id unique, as this component is shown on the page 3 times at the same time, so using the same ID for all of them will result in the wrong label getting referenced.

Reply:
This should be unique since it's taking the label prop but it should probably be somewhat sanitized (ex: spaces removed)